meetings/2018-01-10.md

```diff
@@ -0,0 +1,117 @@
+   * Rich: concern that there is a real focus on feedback to TSC, may be sending wrong signals
+     about decision making (most decisions are not made by the TSC), and many of the TSC
+     members were surprised as the minutes were written in a way that suggested that the TSC
+     was being spoken for when most TSC members weren't aware of this initiative 
+   * Fishrock: TLDR, important to make sure it is clarified in doc.
+   * James: purely a channel for that group to indicate what things are a priority
+   * Franziska: since its a CommComm initiative, how about less focus in the charter on the TSC
+   * Agreed to take back to GitHub and discuss there.
+
+* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)
+  * N-API
+    * Making progress on satisfying criteria to exit experimental status.
+      * Bcrypt ported and being used in production soon
+      * weak-napi published as alternative to node-weak
+      * Module used in dev-edge production site ported and working to have it be used in
+        production there
+      * Working with maintainer of node-pre-gyp to define/document how to use N-API modules
+        with it.
+    * Moderation
+      * One user permanently banned for wiki vandalism.
+      * Myles dealt with a spam-bot invasion on #node-dev IRC channel
+      * One user.s comment was moderated by a Collaborator, moderation group called into 
+        advise. User was also emailed by a moderation team member. Conversation
+        seems to have righted itself, and user is getting necessary help in the correct forum.
+      * We.ve received two or so blank emails to the CoC reporting address.
+    * Streams+
+      * Fishrock: Ironing out some goals, more prototyping
+      * Matteo: refactoring of readable module landed to address long-standing issue, please
+         test/verify
+    * http2, significant updates, hardening, want to have it out of experimental by 10
+    * Modules: Not too much new since last update. Some investigation on module specifier.
+      dynamic import landed on master, looking backport to 9. More in-depth update next week.
+   * Workers: nothing new right now. 
+   * Error Messages: James, progress being made, most on javascript land are complete.  A
+     good number on that side.  Discuss on moving to javascript or not.  Goal is to have done
+     in time for 10.x.  
+   * Core Promise APIs - fs PR there, was waiting on some of the error checking. 
+     waiting for those to land to push forward.  Goal is to take in progress label off. 
+     Matteo has been working on using async generators with streams.
+   * Async Hooks: Some help getting changes for Async hooks for Nan would be good.
+   * V8 currency: Nothing new
+   * TSC governance, Myles will be reviving this and initial focus may be on providing growth
+    paths for contributors.  Refactoring on strategic initiatives to include .stakeholders.
```

Comment:
Are these supposed to be separate lines/sub-bullet points or something?

Reply:
@Trott, they could be but not sure they have to be. Myles mentioned 2 things that he would be reviewing that issue and work that would be started would have an initial focus on providing growth for contributors.  The other thing he mentioned is that he had a PR oven to refactor the list of strategic initiatives to include stakeholders in addition to the champions.   Notes were a bit terse.